ocaml/opam/Portfile

```diff
@@ -62,9 +59,12 @@ if {[string match *gcc* ${configure.compiler}]} {
                     --without-mccs
 }
 
-if {${os.platform} eq "darwin" && ${os.major} < 11} {
-    # Matching ocaml 1.1 PG setting.
-    # See: https://github.com/ocaml/dune/issues/8941
-    build.env-append \
-                    DUNE_CONFIG__COPY_FILE=portable
+if {${os.platform} eq "darwin" && ${os.major} < 14} {
+    known_fail      yes
```

Comment:
Well, no, this is not something acceptable.

Let me find how Opam passes linker flags, and I will make a patch.

Reply:
As you wish. I do not personally have more time to spend on this.

Reply:
I do not think there is a pressing need to update `opam` this very moment and break it for several systems. 2.2.0 has been working fine so far, AFAIK.